### PR TITLE
Do not use height/width when construct polygon

### PR DESCRIPTION
**Pull request by eddyxu**
Height / width are irrelevant to polygon, and datasets like coco do not have height/width with the segmentations.

### CODE REVIEW

File: python/rikai/types/geometry.py [hunk 1]
```diff
@@ -514,7 +514,9 @@ def __init__(
         width: Optional[int] = None,
         mask_type: Mask.Type = Type.POLYGON,
     ):
-        if height is None or width is None:
+        if mask_type != Mask.Type.POLYGON and (
+            height is None or width is None
+        ):
             raise ValueError(
                 "Must provide height and width for RLE or Polygon type"
```
Comment (changhiskhan) [inline python/rikai/types/geometry.py:521]: remove "or Polygon type"?

File: python/rikai/types/geometry.py [hunk 2]
```diff
@@ -572,9 +574,7 @@ def from_polygon(data: list[list[float]], height: int, width: int) -> Mask:
         width: int
```
Comment (changhiskhan) [inline python/rikai/types/geometry.py:574]: remove the docs?

**Review by changhiskhan — APPROVED** [review]
docstrings

File: python/rikai/types/geometry.py (hunk 2 above)
Comment (eddyxu) [reply python/rikai/types/geometry.py]: fixed

File: python/rikai/types/geometry.py (hunk 1 above)
Comment (eddyxu) [reply python/rikai/types/geometry.py]: fixed

**Pull request merged**